pkgs/os-specific/linux/audit/default.nix

```diff
@@ -18,20 +18,14 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "audit";
-  version = "3.1.1";
+  version = "3.1.2";
 
   src = fetchurl {
     url = "https://people.redhat.com/sgrubb/audit/audit-${finalAttrs.version}.tar.gz";
-    hash = "sha256-RuRrN2I8zgnm7hNOeNZor8NPThyHDIU+8S5BkweM/oc=";
+    hash = "sha256-wLF5LR8KiMbxgocQUJy7mHBZ/GhxLJdmnKkOrhA9KH0=";
   };
 
-  patches = [
-    ./000-fix-static-attribute-malloc.diff
-    ./001-ignore-flexible-array.patch
-  ];
-
   postPatch = ''
-    sed -i 's,#include <sys/poll.h>,#include <poll.h>\n#include <limits.h>,' audisp/audispd.c
```

Comment:
What is the reasoning for removing this sed? I cannot find what changed to allow its removal.

Reply:
Hello, thanks for the review!

This regex does not match anything since v3 with this commit (from nov. 2018):

https://github.com/linux-audit/audit-userspace/commit/6d6c65e8e374ce31037e20b1cdc314808efd0e3c#diff-035369519d0ce9cf625d86e351f8a1f5d8dc0609349b0775f30d82b30418a0d5L34

Reply:
Ah i see. I am curious for why it was changed, as there does exist two other cases of `<sys/poll.h>` in other files, but are both 15 year old and not recent.
- https://github.com/linux-audit/audit-userspace/blob/d1ccd8f642e11ad65cdc8c7091e8ef0304a3fd3d/lib/netlink.c#L30
- https://github.com/linux-audit/audit-userspace/blob/d1ccd8f642e11ad65cdc8c7091e8ef0304a3fd3d/lib/netlink.c#L30

Reply:
From a git blame, i can see that the line was added for presumably musl support. https://github.com/NixOS/nixpkgs/commit/32b472dd9d69f7cb5fff32058767c72d0d9799e3